src/employee/quiz.md

```diff
@@ -4,33 +4,36 @@ Welcome at VI! To get you going we want to check your know-how. So let's start w
 > Goal: We know where you stand and how to proceed with your work at VI.
 
 ---
+## Development Basics
 ### Git
 
 <questions-component>
   <question-component>What's Git?</question-component>
-  <answer-component>Git is a versioning software which allows developers to manage different versions of their code. It is like a time machine, which allows you to go back in time and merge different timelines together.</answer-component>
+  <answer-component>Git is a versioning software which allows users to manage different versions of their code. It is like a time machine, which allows you to go back in time and merge different timelines together.</answer-component>
+  <question-component>What's the difference between Git and GitHub?</question-component>
+  <answer-component>Git is the tool, GitHub is the platform. On GitHub you can host repositories.</answer-component>
```

Comment:
At this point the user may not know what a repository is. (It's part of one of the next questions)